app/src/main/java/de/xikolo/controllers/dialogs/HelpdeskTopicAdapter.kt

```diff
@@ -79,4 +101,10 @@ class HelpdeskTopicAdapter(private val onTopicClickedListener: OnTopicClickedLis
         )
     }
 
+    private fun getCourseYear(course: Course?): Int {
+        val calendar = Calendar.getInstance()
+        calendar.time = course!!.startDate
+        return calendar.get(Calendar.YEAR)
+    }
+
```

Comment:
This would fit perfectly inside `DateExtensions`. It could then be used like `course.startDate.year`